From 1cbed14b39f698c3b6339905ee19cc35389232e3 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Tue, 21 Jun 2022 23:04:52 +0200
Subject: [PATCH 309/417] arm64: dts: rockchip: rk3399-pinephone-pro: Cleanup
 regulator sleep states

Forcing SD card I/O voltage to 3.3V in suspend is probably not a great
idea. A lot of properties were superfluous, etc.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 .../dts/rockchip/rk3399-pinephone-pro.dts     | 24 ++-----------------
 1 file changed, 2 insertions(+), 22 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 182e042a1..6c5ce176b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -410,10 +410,6 @@ vcc5v0_typec: vcc5v0-typec-regulator {
 		regulator-max-microvolt = <5000000>;
 		regulator-name = "vcc5v0_typec";
 		vin-supply = <&boost_otg>;
-
-		regulator-state-mem {
-			regulator-on-in-suspend;
-		};
 	};
 
 	/* Main 3.3v supply */
@@ -425,10 +421,6 @@ vcc3v3_sys: wifi_bat: vcc3v3-sys {
 		regulator-min-microvolt = <3300000>;
 		regulator-max-microvolt = <3300000>;
 		vin-supply = <&vcc_sysin>;
-
-		regulator-state-mem {
-			regulator-on-in-suspend;
-		};
 	};
 
 	vcc_4g_5v: vcc-4g-5v {
@@ -649,7 +641,6 @@ vcc_1v8: vcc_wl: DCDC_REG4 {
 				regulator-max-microvolt = <1800000>;
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <1800000>;
 				};
 			};
 
@@ -660,7 +651,7 @@ vcca3v0_codec: LDO_REG1 {
 				regulator-min-microvolt = <3000000>;
 				regulator-max-microvolt = <3000000>;
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
 				};
 			};
 
@@ -682,7 +673,7 @@ vcca1v8_codec: LDO_REG3 {
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
 				};
 			};
 
@@ -694,7 +685,6 @@ vcc_power_on: LDO_REG4 {
 				regulator-max-microvolt = <3300000>;
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <3300000>;
 				};
 			};
 
@@ -706,7 +696,6 @@ vcc_3v0: LDO_REG5 {
 				regulator-max-microvolt = <3000000>;
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <3000000>;
 				};
 			};
 
@@ -718,7 +707,6 @@ vcc_1v5: LDO_REG6 {
 				regulator-max-microvolt = <1500000>;
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <1500000>;
 				};
 			};
 
@@ -726,9 +714,6 @@ vcc1v8_dvp: LDO_REG7 {
 				regulator-name = "vcc1v8_dvp";
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
-				regulator-state-mem {
-					regulator-off-in-suspend;
-				};
 			};
 
 			vcc3v3_s3: LDO_REG8 {
@@ -746,10 +731,6 @@ vccio_sd: LDO_REG9 {
 				regulator-name = "vccio_sd";
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <3300000>;
-				regulator-state-mem {
-					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <3300000>;
-				};
 			};
 
 			vcc3v3_s0: SWITCH_REG {
@@ -769,7 +750,6 @@ boost_otg: DCDC_BOOST {
 				regulator-max-microvolt = <5000000>;
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
-- 
2.35.3

